### ISSUE TITLE
at exactly which checkpoint(s) (one with best validation set result) low resource number are reported in paper

### ISSUE THREAD

**Issue by agenius5** [google-research/pegasus#137]
Hi @JingqingZ 

I wanted to know at which checkpoint low resource number were obtained. I know it's the checkpoint with best validation perfomarnce and it's not necessarily 2000 steps. I can't replicate the fine-tuning process with similar hyperparameters. So please share those the best validation checkpoint details for 1k and 10k examples for varoius datasets. I don't want the checkpoint files, just the numbers. It'll be a great help.

**Comment by JingqingZ** [comment]
Hi, I am sorry unfortunately I can't trace back to the checkpoints and exact number of steps we used, besides the information we reported in the paper.
